content/4_Gold/BFS.problems.json

```diff
@@ -41,6 +41,19 @@
         "kind": "internal"
       }
     },
+    {
+      "uniqueId": "usaco-491",
+      "name": "Piggyback",
+      "url": "http://www.usaco.org/index.php?page=viewproblem2&cpid=491",
+      "source": "Old Silver",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["BFS"],
+      "solutionMetadata": {
+        "kind": "internal",
+        "usacoId": "491"
```

Comment:
??

Reply:
moved a problem

Reply:
`usacoId` isn't necessary for an `internal` solution.

if the solution is internal, remove it.

otherwise, change `kind`